2_second_hw/2_second_task.py

```diff
@@ -0,0 +1,52 @@
+"""
+You have 2 companies with people. One of the companies, let it be Eleks, was taken over by Toshiba. Show it in code.
+Keep in mind that people with the same name can be in both companies
+"""
+eleks = ['Geralt', 'Triss', 'Lutic']
+toshiba = ['Yennefer', 'Ciri', 'Vesemir', 'Geralt']
+toshiba.extend(eleks)
+print(toshiba)
+
```

Comment:
Eleks still have employees. Please fix this

Reply:
Fixed, please check